rules/DeadCode/Rector/Node/RemoveNonExistingVarAnnotationRector.php

```diff
@@ -74,27 +61,20 @@ public function get()
      */
     public function getNodeTypes(): array
     {
-        return [
-            Foreach_::class,
-            Static_::class,
-            Echo_::class,
-            Return_::class,
-            Expression::class,
-            Throw_::class,
-            If_::class,
-            While_::class,
-            Switch_::class,
-            Nop::class,
-        ];
+        return [Stmt::class];
```

Comment:
checking `@var` on every Stmt probably slowing down performance, as it check everything except in `shouldSkip()` method; , the `Namespace`, `ClassLike`, `FunctionLike` mostlikely not needed check